### ISSUE TITLE
Pre-processing

### ISSUE THREAD

**Issue by TheKOne** [ndkhanh360/CAER#5]
Hello!,Could you fix the results of dlib's cnn detector for the training set and test set?thank you so much! Meanwhile,Could you upload your trained models?

**Comment by ndkhanh360** [comment]
Sorry for the late response, 
I've modified the readme with updated dlib's results and pretrained model information. Please have a check!

**Comment by TheKOne** [comment]
Hello,Sir! I have downloaded your pre-trained model caers.pth and saved it in the saved/models directory. I would like to ask whether the configuration file can be named config.json and stored in the configs directory, namely configs/config.json, test When I typed in the console: python --config configs/config.json --resume saved/models/caers.pth, but there was an error. May I ask where is the problem with my thinking?
